services/efs-guardian/src/simcore_service_efs_guardian/services/efs_manager.py

```diff
@@ -36,5 +40,12 @@ async def create_project_specific_data_dir(
             / f"{node_id}"
             / f"{storage_directory_name}"
         )
+        # Ensure the directory exists with the right parents
         Path.mkdir(_dir_path, parents=True, exist_ok=True)
+        # Change the owner to user id 8006(efs) and group id 8106(efs-group)
+        os.chown(_dir_path, _EFS_LINUX_USER, _EFS_GROUP_LINUX_GROUP)
+        # Set directory permissions to allow group write access
+        os.chmod(
+            _dir_path, 0o770
```

Comment:
Rewrite `0o770` using `stat` constants instead

```python
import os
import stat

os.chmod(_dir_path, stat.S_IRUSR | stat.S_IWUSR | stat.S_IXUSR | stat.S_IRGRP | stat.S_IWGRP | stat.S_IXGRP)
```

much better that all these comments :-)

Reply:
to be honest for me 770 is much more readable than the stat thing, also the example you provided does not create 770. If that is OK with you I would leave the 770 like this for now. I have improved all the other review comments provided.